test/app-tap/module_api.c

```diff
@@ -3045,6 +3058,7 @@ luaopen_module_api(lua_State *L)
 		{"isdecimal", test_isdecimal},
 		{"isdecimal_ptr", test_isdecimal_ptr},
 		{"box_schema_version_matches", test_box_schema_version},
+		{"box_session_id", test_box_session_id},
```

Comment:
Nit: Please rename "box_session_id" to "box_session_id_matches" here for consistency with "box_schema_version_matches".

Reply:
Fixed: rename all test related names similar to `box_schema_version`.